scipy/spatial/_spherical_voronoi.py

```diff
@@ -184,8 +184,8 @@ def __init__(self, points, radius=1, center=None, threshold=1e-06):
         # test degenerate input
         self._rank = np.linalg.matrix_rank(self.points - self.points[0],
                                            tol=threshold * self.radius)
-        if self._rank <= 1:
-            raise ValueError("Rank of input points must be at least 2")
+        if self._rank < self._dim:
+            raise ValueError("Rank of input points must be at least {0}".format(self._dim))
```

Comment:
Can you remind me--this will catch both equatorial and non-equatorial circular input, right?

So in both of those cases the user will basically get an exception from us/Python rather than a dump from Qhull?

Reply:
Right on both points.